alacritty.yml

```diff
@@ -33,8 +33,24 @@ window:
   # Setting this to false will result in window without borders and title bar.
   decorations: true
 
-# How many lines of scrollback to keep
-scroll_history: 10000
+scrolling:
+  # How many lines of scrollback to keep,
+  # '0' will disable scrolling.
+  history: 10000
+
+  # Number of lines the viewport will move for every line
+  # scrolled when scrollback is enabled (history > 0).
+  multiplier: 1
```

Comment:
`3` ?

Reply:
I think `1` would feel unusually slow for a lot of people. 3 seems to be a good value, some terminals scroll even a lot more. Termite scrolls for 7 for example.

Thanks for pointing out this mistake though. :)